includes/discovery/sensors/ipmi.inc.php

```diff
@@ -64,6 +61,35 @@
         }
     }
 
+    $nmClient = new NodeManager($client);
+    if ($nmClient->isPlatformSupported()) {
+        // Set Node Manager connection properties.
+        foreach ($nmClient->discoverAttributes() as $nmAttribKey => $nmAttribValue) {
+            set_dev_attrib($device, "node_manager_$nmAttribKey", $nmAttribValue);
+        }
+
+        $ipmi_unit_type = Config::get('ipmi_unit.Watts');
```

Comment:
ipmi_unit.Watts should be lowercase and added to config_definitions.  Same with other units, should be strtolower() to make them consistent.